components/scream/src/physics/shoc/atmosphere_macrophysics.hpp

```diff
@@ -2,7 +2,7 @@
 #define SCREAM_SHOC_MACROPHYSICS_HPP
 
 #include "share/atm_process/atmosphere_process.hpp"
-#include "share/scream_parameter_list.hpp"
+#include "ekat/scream_parameter_list.hpp"
```

Comment:
This is one of those renames that's being saved for a later PR, right?

Reply:
Right.